contract/src/internals/helpers.rs

```diff
@@ -53,28 +53,30 @@ pub(crate) fn assert_valid_drop_config(drop_config: &Option<DropConfig>) {
             "Cannot have 0 uses per key for a drop config"
         );
 
-        // Assert that if the claim_interval is some, the start_timestamp is also some
-        assert!(
-            (config.claim_interval.is_some() && config.start_timestamp.is_none()) == false,
-            "If you want to set a claim interval, you must also set a start timestamp"
-        );
-
-        // Assert that both the start_timestamp and end timestamps are greater than the current block
-        assert!(
-            config.start_timestamp.unwrap_or(env::block_timestamp()) >= env::block_timestamp(),
-            "The start timestamp must be greater than the current block timestamp"
-        );
-        assert!(
-            config.end_timestamp.unwrap_or(env::block_timestamp()) >= env::block_timestamp(),
-            "The end timestamp must be greater than the current block timestamp"
-        );
+        if let Some(time_data) = &config.time {
```

Comment:
what's the reason to name it `time_data` internally?

Reply:
No reason - can rename to time_config or simply `data`